### ISSUE TITLE
The provided pre-trained PointPillar test metrics on the kitti test set differed significantly from those in the paper

### ISSUE THREAD

**Issue by MLbeginner2** [open-mmlab/OpenPCDet#1160]
I submitted the official OpenPCDet results of pointpillar_7728.pth on the KITTI test set to KITTI, but the metrics are far below the values in the paper, is it because OpenPCDet's PointPillar only used 3712 samples during training? Do I need to change any parameters if I want to train with all the training samples?
The following figure shows the results of PointPillar on the KITTI test set：
![image](https://user-images.githubusercontent.com/45509031/198594049-8f51b63a-9a36-482f-882e-204d4fc388b1.png)

**Comment by Townjj** [comment]
It's not official code of Pointpillar.

**Comment by bqcs** [comment]
> I submitted the official OpenPCDet results of pointpillar_7728.pth on the KITTI test set to KITTI, but the metrics are far below the values in the paper, is it because OpenPCDet's PointPillar only used 3712 samples during training? Do I need to change any parameters if I want to train with all the training samples? The following figure shows the results of PointPillar on the KITTI test set： ![image](https://user-images.githubusercontent.com/45509031/198594049-8f51b63a-9a36-482f-882e-204d4fc388b1.png)

Have you identified the cause of the problem?
